src/const/message.js

```diff
@@ -0,0 +1,9 @@
+export const message = {
+	START: '숫자 야구 게임을 시작합니다.',
+	END: '3개의 숫자를 모두 맞히셨습니다! 게임 종료',
+	RESTART: '게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요. \n',
+	STRIKE: '스트라이크',
+	BALL: '볼',
+	NOTHING: '낫싱',
+	INPUT: '숫자를 입력해주세요 : ',
+}
```

Comment:
```suggestion
export const MESSAGE = Object.freeze({
	START: '숫자 야구 게임을 시작합니다.',
	END: '3개의 숫자를 모두 맞히셨습니다! 게임 종료',
	RESTART: '게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요. \n',
	STRIKE: '스트라이크',
	BALL: '볼',
	NOTHING: '낫싱',
	INPUT: '숫자를 입력해주세요 : ',
})
```
타입스크립트면 `as const` 사용할 것 같은데 바닐라 JS 환경이라서 `Object.freeze()`를 사용해도 좋을 것 같아요!